metrics-operator/controllers/common/providers/dynatrace/dynatrace.go

```diff
@@ -37,7 +38,8 @@ type DynatraceData struct {
 
 // EvaluateQuery fetches the SLI values from dynatrace provider
 func (d *KeptnDynatraceProvider) EvaluateQuery(ctx context.Context, metric metricsapi.KeptnMetric, provider metricsapi.KeptnMetricsProvider) (string, []byte, error) {
-	qURL := provider.Spec.TargetServer + "/api/v2/metrics/query?metricSelector=" + metric.Spec.Query
+	baseURL := d.normalizeAPIURL(provider.Spec.TargetServer)
+	qURL := baseURL + "v2/metrics/query?metricSelector=" + metric.Spec.Query
```

Comment:
can we move the string to a constant ?

Reply:
it is not reused anywhere else, I don't think we need to expose this info to the whole package. It is only valid for Dynatrace metrics v2 APIs and not for DQL.